umbra/controller.py

```diff
@@ -294,17 +295,37 @@ def browse_page_sync():
                 message.ack()
             except brozzler.ShutdownRequested as e:
                 self.logger.info("browsing did not complete normally, requeuing url {} - {}".format(url, e))
-                message.requeue()
+                message.requeue()  # republish?
```

Comment:
Barbara and I discussed. I think I understand the situation here:
a) the difference between requeueing vs republishing is adding the URL to the top (requeue) or the bottom (republish) of the queue
b) umbra uses brozzler code
b) This happens at shutdown time, and it is probably fine to put the urls at the top of the stack at that time.

Reply:
That is, this is not a blocker. Merging.